actions/jib-uploader/action.yaml

```diff
@@ -0,0 +1,29 @@
+name: "Build and upload Jib "
+description: "Build the Jib image and upload it to the image repository"
+
+inputs:
+  image:
+    description: "The image name with its repository name"
+    required: true
+  tags:
+    description: "The tags to use to release the image"
+    required: true
+  class:
+    description: "The entrypoint class to be used for the image"
+    required: true
+  working-directory:
+    description: "working directory to run the commands"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build Docker image
+      run: |
+        ./gradlew jib \
+          --info --stacktrace \
+          --image=${{ inputs.image }} \
+          -Djib.container.mainClass=${{ inputs.class }} \

```

Comment:
Is this really needs to be required?
It think the user can configure this in its own application.

Reply:
I am not sure to be honest, should I remove it ?

Reply:
should we still support it as an optional feature or should we remove it completely? what do you guys think ?

Reply:
I would have it removed. The user should take of the mainClass 🙂

Reply:
We could make it optional as well if there are multiple jib images to be built with different main classes. But I don't have such a use case so I would be fine with removing it

Reply:
@philipp94831 following up this. But how would it work when you have multiple entry points in every module of your project ? do we have such thing atm ? or it is not something it happens quite often ? if not, then I would just remove it.

Reply:
User needs to configure mainClass in gradle file

Reply:
Ok then I will remove it